### PR TITLE
Server verification test

### PR DESCRIPTION
**Pull request by forgetso**

### CODE REVIEW

File: demos/client-example-server/src/controllers/auth.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ const signup = async (
         if (dbUser) {
             return res.status(409).json({ message: 'email already exists' })
         }
-
+        console.log('payload', payload)
```
Comment (goastler) [inline demos/client-example-server/src/controllers/auth.ts:56]: intentional debug?

File: demos/cypress-shared/cypress/e2e/correct.captcha.cy.ts [hunk 2]
```diff
@@ -30,7 +30,6 @@ describe('Captchas', () => {
                 context: { datasetWithSolutionHashes },
             })
         }
-        cy.intercept('/dummy').as('dummy')
```
Comment (goastler) [inline demos/cypress-shared/cypress/e2e/correct.captcha.cy.ts:33]: why has this been removed?

File: packages/cli/src/cli.ts [hunk 3]
```diff
@@ -32,6 +32,16 @@ async function main() {
         unsolved: { count: 0 },
     })
 
+    if (config.devOnlyWatchEvents) {
```
Comment (goastler) [inline packages/cli/src/cli.ts:35]: nice!

**Review by goastler — CHANGES_REQUESTED** [review]

File: demos/client-example-server/src/controllers/auth.ts (hunk 1 above)
Comment (forgetso) [reply demos/client-example-server/src/controllers/auth.ts:56]: Yep, this is to help identify what's gone wrong when you're running locally. This is just a demo package.

File: demos/cypress-shared/cypress/e2e/correct.captcha.cy.ts (hunk 2 above)
Comment (forgetso) [reply demos/cypress-shared/cypress/e2e/correct.captcha.cy.ts:33]: I think it was just copied from stackoverflow by mistake. We don't have any API calls with `dummy` in them!

**Review by goastler — DISMISSED** [review]

**Review by goastler — APPROVED** [review]

**Pull request merged**